phylogenetic/rules/merge_inputs.smk

```diff
@@ -0,0 +1,184 @@
+"""
+This part of the workflow merges inputs based on what is defined in the config.
+
+OUTPUTS:
+
+    metadata  = "results/metadata.tsv"
+    sequences = "results/sequences.fasta"
```

Comment:
(nit) Match formatting used elsewhere


```suggestion
    metadata  = results/metadata.tsv
    sequences = results/sequences.fasta
```

Reply:
Updated in 8b25198b89adda63265143abdb0168d893fe1555